lib/skate/detours/detours.ex

```diff
@@ -352,18 +357,27 @@ defmodule Skate.Detours.Detours do
          %Detour{} = detour,
          true
        ) do
+    update_swiftly(detour, :delete, true)
+  end
+
+  defp update_swiftly(%Detour{status: :active} = detour, :create, true) do
+    {:ok, adjustment_request} = Swiftly.API.Requests.to_swiftly(detour)
+
+    service_adjustments_module().create_adjustment_v1(
+      adjustment_request,
+      build_swiftly_opts()
+    )
+  end
+
+  defp update_swiftly(%Detour{status: :past} = detour, :delete, true) do
```

Comment:
suggestion: is it still useful to keep the third argument that's always `true` here? Or could we simplify this and drop these instances of `update_swiftly/3` to `update_swiftly/2` calls?